### DISCUSSION TITLE
Literally nowhere obvious on a google search where we goto to get bids on how much FIL for our data - to store

### DISCUSSION THREAD

**Discussion by johnda98** [filecoin-project/community#600] [General] [answered]
Literally nowhere obvious on a google search where we goto to get bids on how much FIL for our data - to store

any hints on where to go ?   web3.storage..  nope they want to know the color of my underpants..

is there a site..maketspace of node hosters that r proven and accept content for filcoin?

like a mystery to a layperson ... WHERE does one go to get a FIL price to store 5MB or less..  say a gatsby static site

**Comment by zklim** [comment]
I wanted to know how storage and retrieval costs calculated too. And is there any minimum size to commit and other requirements like deals collateral.

**Reply by johnda98** [reply]
i just want to pay anyone who runs a node to store a simple gatsby site and pay them in FIL.. .without having to tell them the color of my pants

**Comment by johnda98** [comment, marked as answer]
https://docs.filecoin.io/get-started/store-and-retrieve/store-data/
